packages/mask/src/plugins/Avatar/SNSAdaptor/NFTAvatar.tsx

```diff
@@ -125,13 +138,17 @@ export function NFTAvatar(props: NFTAvatarProps) {
         WalletMessages.events.selectProviderDialogUpdated,
     )
 
-    const LoadStatus = Array.from({ length: 8 })
-        .fill(0)
-        .map((_, i) => (
-            <div key={i} className={classes.skeletonBox}>
-                <Skeleton animation="wave" variant="rectangular" className={classes.skeleton} />
-            </div>
-        ))
+    const LoadStatus = (
+        <List className={classes.list}>
+            {Array.from({ length: 8 })
+                .fill(0)
+                .map((_, i) => (
```

Comment:
`import { range } from 'lodash-unified'`
```suggestion
            {range(8).map((i) => (
```